jnigen/README.md

```diff
@@ -1,6 +1,6 @@
 # Experimental generator for FFI+JNI bindings.
 
-This package consists of the code generator `package:jnigen` and a few [example](examples).
+This package consists of the code generator `package:jnigen` and a few [example](example).
```

Comment:
"examples" should be plural in the sentence, and the link "example": `[examples](example).`

Reply:
Right, I'll fix that. Thanks!